frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -158,7 +158,7 @@ export default function CohortsPage({
             },
           } as SampleWhere
         }
-        samplesRefetchWhereVariables={(samplesParsedSearchVals: string[]) => {
+        samplesRefetchWhereVariables={(samplesParsedSearchVals) => {
```

Comment:
Specifying the type here was redundant because it's inferred automatically by the IDE based on `IRecordsListProps` in `RecordsList.tsx`.